docs/hotkeys.md

```diff
@@ -34,11 +34,11 @@
 |Narrow to the stream of the current message|<kbd>s</kbd>|
 |Narrow to the topic of the current message|<kbd>S</kbd>|
 |Narrow to compose box message recipient|<kbd>Meta</kbd> + <kbd>.</kbd>|
-|Narrow to a topic/direct-chat, or stream/all-direct-messages|<kbd>z</kbd>|
-|Narrow to all messages|<kbd>a</kbd> / <kbd>Esc</kbd>|
-|Narrow to all direct messages|<kbd>P</kbd>|
-|Narrow to all starred messages|<kbd>f</kbd>|
-|Narrow to messages in which you're mentioned|<kbd>#</kbd>|
+|Zoom in/out the message's conversation context|<kbd>z</kbd>|
+|Go to your combined feed|<kbd>a</kbd> / <kbd>Esc</kbd>|
+|Go to your direct message feed|<kbd>P</kbd>|
+|Go to your starred messages|<kbd>f</kbd>|
+|Go to messages in which you're mentioned|<kbd>#</kbd>|
```

Comment:
I'm not entirely sure about the language we should use here, but should we consider using "Go to" instead of "Narrow" for lines 34-36 as well? To me, "Go to" seems clearer than "Narrow." However, this is just my opinion, so we'll need to wait for feedback from Neil and the mentors.

The rewording of the zoom in/out feature seems great, though!

Reply:
Thanks for the review!

Ah, good point! Yes, I was considering the same, but I wasn't sure because those narrowing commands kind of use the current context and don't switch to something completely different or a named narrow, like the menu buttons do. But, it's probably just me.
And that's also why line 41 doesn't feel like a great fit either, because it's not a named narrow.
Thank you for sharing your thoughts!

Zoom in/out credits: Zulip webapp

Reply:
For context, historically - at least in ZT - we've had confusion over communicating 'a narrow' (message view; noun) and 'narrowing' (switching to a message view; verb).

Most of this likely comes from the phrasing in the code, and also how the API defines a narrow as a filter of the messages you can see: https://zulip.com/api/construct-narrow.
(note we currently use an old API style for narrows, which needs to be refactored to support the documented style which uses a dict)

You can also see this in our user-facing tutorial: https://github.com/zulip/zulip-terminal/blob/main/docs/getting-started.md#Narrowing

So, there's potentially some small follow-up we could do to clarify this for end-users, if we're switching to 'message view' for the noun form.

Regarding the specific verb prefix selection (go-to vs narrow):
- ideally it would be as concise as possible, as well as consistent
- some of the contextual grouping we have was originally intended to make it easier to understand based on the category titles, but since the descriptions can appear without them, then we can't assume their presence and simplify that much
- 'go' is used a lot in the web app descriptions for 'narrowing' (https://zulip.com/help/keyboard-shortcuts)
- the web app or docs actually don't seem to use narrow for either form? (outside of API)
- narrowing can change keyboard focus, but we currently use 'go' for the keyboard movement, so that could be confusing when really it's specifically changing the message view

Reply:
Re `z`, this is a simple toggleable hotkey in ZT, which we introduced before the web app switched to using just `s` for doing *almost* the same. In web it doesn't apply to DMs, so it's not really a 'safe' toggle between views - and I'm not sure if `shift s` (`S`) still works. I'm inclined to suggest Toggle could be a good phrase here.

'Zoom in/out' is definitely cleaner, and relates to why we chose `z` in the first place, but the challenge with the phrasing here has always been how to cleanly describe the different contexts. 'Conversation' seems to be the term used for 'most zoomed in' view at the moment, but DM feed and channel feeds are not quite as clear cut, but might be useful?

(for the reasons above, `z` should be right next to `s`/`S` in the help)